### ISSUE TITLE
Can't scroll when editing work pool template

### ISSUE THREAD

**Issue by madkinsz** [PrefectHQ/prefect#9128]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I refreshed the page and this issue still occurred.
- [X] I checked if this issue was specific to the browser I was using by testing with a different browser.

### Bug summary

I cannot scroll from within the JSON editor which makes it very challenging to edit

### Reproduction

Open the work pool JSON editor
Attempt to scroll with mouse inside editor — does not work
Move mouse outside editor and attempt to scroll — does work

### Error

n/a

### Browers

- [x] Chrome
- [X] Firefox
- [ ] Safari
- [ ] Edge

### Prefect version

```Text
Version:             2.9.0+76.g097a4819b.dirty
API version:         0.8.4
Python version:      3.11.2
Git commit:          097a4819
Built:               Thu, Apr 6, 2023 1:53 PM
OS/Arch:             darwin/arm64
Profile:             default
Server type:         ephemeral
Server:
  Database:          sqlite
  SQLite version:    3.39.5
```

**Comment by madkinsz** [comment]
I'll try to reproduce on Chrome when I have time

**Comment by znicholasbrown** [comment]
Resolved by: https://github.com/PrefectHQ/prefect-design/pull/722

**Comment by znicholasbrown** [comment]
Fix merged - will go out with the next release of prefect-design... I'll close this when that bump makes it here

**Comment by zhen0** [comment]
Should be resolved in prefect design 1.7.1 and the latest oss and cloud released
